scripts/core/game.gd

```diff
@@ -91,14 +125,15 @@ func _ready():
 	## Duplicate cards
 	card_theme += card_theme  # Make a set of cards (Duplicate them)
 	card_theme.shuffle()  # Shuffle cards for the game
```

Comment:
On a line 122 there is a shuffle that is obsolete. Commenting on this line because i can't comment on the previous one.

Reply:
Nevermind. Realised that that line is used to shuffle which cards are going to appear on the board, not where.